partcad/src/partcad/shape.py

```
@@ -506,20 +529,43 @@
+            await runtime.ensure_async("cadquery==2.4.0")
+            response_serialized, errors = await runtime.run_async(
+                [
+                    wrapper_path,
+                    os.path.abspath(filepath),
+                ],
+                request_serialized,
+            )
+            sys.stderr.write(errors)
+
+            response = base64.b64decode(response_serialized)
+            result = pickle.loads(response)
```

Ruff (0.8.2)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)